packages/common/postgres_document_store.py

```
@@ -0,0 +1,262 @@
+
+        Raises:
+            KeyError: If document not found.
+        """
+        with self.conn.cursor() as cur:
+            cur.execute(
+                "SELECT content FROM document_content WHERE doc_id = %s", (str(doc_id),)
+            )
+            row = cur.fetchone()
+
+        if not row:
+            raise KeyError(f"Document content not found for {doc_id}")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)